std/http/io_test.ts

```diff
@@ -343,6 +343,44 @@ test("writeResponse with trailer", async () => {
   assertEquals(ret, exp);
 });
 
+test("writeResponse with repeating headers", async () => {
+  const body = "Hello";
+
+  const res: Response = { body, headers: new Headers(
+    [
+      ["set-cookie", "user.session=qwertz; Max-Age=86400"],
+      [
+        "set-cookie",
+        "mykey=myvalue; expires=Mon, 17-Jul-2017" +
+        " 16:06:00 GMT; Max-Age=31449600; Path=/; secure",
+      ],
+      ["set-cookie", "b=456; Domain=example.com; Secure; HttpOnly"],
+      ["x-other-header", "some, value"],
+      ["x-other-header", "custom=pair; values"],
+    ],
+  ) };
+
+  const w = new Buffer();
+  await writeResponse(w, res);
+
+  const ret = w.toString();
+  const exp = [
+    "HTTP/1.1 200 OK",
+    `content-length: ${body.length}`,
+    "set-cookie: user.session=qwertz; Max-Age=86400",
+    "set-cookie: mykey=myvalue; expires=Mon, 17-Jul-2017" +
+    " 16:06:00 GMT; Max-Age=31449600; Path=/; secure",
+    "set-cookie: b=456; Domain=example.com; Secure; HttpOnly",
+    "x-other-header: some, value, custom=pair; values",
+    "",
+    "5",
+    body,
+    "0",
+    "",
+  ].join("\r\n");
+  assertEquals(ret, exp);
+});
```

Comment:
Nice test, but you've changed a bunch of code elsewhere in this file.

Please run tools/format.js

Reply:
Oh sorry, I thought "deno fmt std/http/io_test.ts" would do it for me. Thanks for the command I searched for :D